gasometer/src/lib.rs

```diff
@@ -610,7 +610,7 @@ pub fn dynamic_opcode_cost<H: Handler>(
 			}
 		}
 
-		_ => GasCost::Invalid,
+		_ => return Err(ExitError::InvalidCode(opcode.0)),
```

Comment:
There are a lot more uses of `GasCost::Invalid` in the gasometer, especially due to config options to support older EVM versions.

I'd suggest we keep `GasCost::Invalid(u8)` and propagate it like it currently does.